ctools/defragment_sharded_collection.py

```diff
@@ -556,6 +556,10 @@ async def get_chunk_size(ch):
 
         return data_size_kb
 
+    async def exec_throttle():
```

Comment:
Rather than sleeping a fix amount of time after every migration I would expose a parameter minMigrationPeriodMillisecs that we can use to cap the frequency of migrations:
```
do {
   start = now()
   migrate()
   duration = now() - start
   if (duration < minMigrationPeriodMillisecs)
      sleep(minMigrationPeriodMillisecs - duration)
} while(needToMigrate)

Reply:
The description for the parameter  would be something like: "Minimum amount of time between subsequent migrations (milliseconds)"